build_artifacts/v2/v2.0/v2.0.0/v2.0.0-beta/dirs/etc/maxdome/kernels/glue/kernel_launcher.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+check_key() {
```

Comment:
Can we just have a method that is more like "export_if_exists" ?

Currently we are adding too many if conditions later in the code. Why not move that if condition here ?